02-peer-protocol.md

```diff
@@ -2518,14 +3046,22 @@ A node:
 
 A receiving node:
   - if `next_funding_txid` is set:
-    - if `next_funding_txid` matches the latest interactive funding transaction:
-      - if it has not received `tx_signatures` for that funding transaction:
+    - if `next_funding_txid` matches the latest interactive funding transaction
+      or the current channel funding transaction:
+      - if `next_commitment_number` is equal to the commitment number of the
+        `commitment_signed` message it sent for this funding transaction:
         - MUST retransmit its `commitment_signed` for that funding transaction.
-        - if it has already received `commitment_signed` and it should sign first,
-          as specified in the [`tx_signatures` requirements](#the-tx_signatures-message):
-          - MUST send its `tx_signatures` for that funding transaction.
+      - if it has already received `commitment_signed` and it should sign first,
+        as specified in the [`tx_signatures` requirements](#the-tx_signatures-message):
+        - MUST send its `tx_signatures` for that funding transaction.
       - if it has already received `tx_signatures` for that funding transaction:
         - MUST send its `tx_signatures` for that funding transaction.
+    - if `next_funding_txid` matches the latest funding transaction:
+      - if that transaction has reached acceptable depth:
```

Comment:
This should be changed to
```
    - if `next_funding_txid` matches the latest funding transaction
      or the matching splice transaction has reached acceptable depth:
```

This is because certain `splice_locked` disconnect flows result in neither side locking the splice into __the latest funding transaction__ but `splice_locked` still needs to be resent during reestablish.

More discussion on this issue is in this thread: https://github.com/ddustin/lightning/pull/6#issuecomment-2610979986